llama_index/vector_stores/chroma.py

```diff
@@ -1,19 +1,33 @@
 """Chroma vector store."""
 import logging
 import math
-from typing import Any, List, cast
+from typing import Any, List, Tuple, cast
 
 from llama_index.data_structs.node import DocumentRelationship, Node
 from llama_index.utils import truncate_text
-from llama_index.vector_stores.types import (
-    NodeWithEmbedding,
-    VectorStore,
-    VectorStoreQuery,
-    VectorStoreQueryResult,
-)
+from llama_index.vector_stores.types import (MetadataFilters,
+                                             NodeWithEmbedding, VectorStore,
+                                             VectorStoreQuery,
+                                             VectorStoreQueryResult)
+from llama_index.vector_stores.utils import (metadata_dict_to_node,
+                                             node_to_metadata_dict)
 
 logger = logging.getLogger(__name__)
 
+def _to_chroma_filter(standard_filters: MetadataFilters) -> dict:
+    """Translate standard metadata filters to Chroma specific spec."""
+    filters = {}
+    for filter in standard_filters.filters:
+        filters[filter.key] = filter.value
+    return filters
+
+def _legacy_metadata_dict_to_node(metadata: dict) -> Tuple[dict, dict, dict]:
+    extra_info = metadata
+    node_info = None
```

Comment:
nit: should this be an empty dict for type checking?

Reply:
+1